lib/sparse/src/common/sparse_vector.rs

```diff
@@ -147,7 +177,7 @@ mod tests {
         assert!(different_length.is_err());
 
         let not_sorted = SparseVector::new(vec![1, 3, 2], vec![1.0, 2.0, 3.0]);
-        assert!(not_sorted.is_err());
+        assert!(not_sorted.is_ok());
```

Comment:
Do we want to hint the user that if their indices are pre-sorted, this indexing operation will be much faster?

Reply:
honestly, sorting 100 integers is a 100 nanosecod operation

Reply:
Aaah. At a few ns*1000 → total a few ms, this is perfectly fine. Thanks!